### PR TITLE
Create Security Team Folder and place holder files for items that need to be added. 

### PR DESCRIPTION
**Pull request by akgraner**

### CODE REVIEW

**Comment by kimwnasptd** [comment]
Thanks for sending the PR @akgraner! Took a small look and had some nits regarding the namings and folder structure.

By being inspired at what we did for the release team https://github.com/kubeflow/community/tree/master/releases what I'd suggest would be:
```
security/
  meetings/
    01-02-2023.md
    01-03-2023.md
    ...
  policies-and-procedures.md
  roadmap.md
  security-team.md
```

The above are minor nits, but I believe will help ensure we use similar names for similar files across the project.

**Comment by akgraner** [comment]
@kimwnasptd thank you! I'll get those fixed. Appreciate the feedback.

**Comment by akgraner** [comment]
delete the original directory and folders and added the directories and folders in the correct format

**Comment by kimwnasptd** [comment]
Thanks!

/lgtm

**Review by eslerm — APPROVED** [review]
LGTM +1

Perhaps add a line to `roadmap.md` and `security-team.md` that describes what they are for.

File: security/meetings/01-18-2023.md
Comment (juliusvonkohout) [inline security/meetings/01-18-2023.md:67]: I think it should be Julius, not Julis

Comment (juliusvonkohout) [inline security/meetings/01-18-2023.md:67]: I think the owners file is missing.

File: security/security-team.md
Comment (james-jwu) [inline security/security-team.md:1]: Empty file?

File: security/README:.md
Comment (james-jwu) [inline security/README:.md:1]: Do you want to add a OWNERS file to give permission to additional folks?

**Comment by juliusvonkohout** [comment]
i recommend to upload the presentation as well 
[Hardening Kubeflow Security for Enterprise Environments - Julius von Kohout, DPDHL & Diana Dimitrova Atanasova, VMware.pdf](https://github.com/kubeflow/kubeflow/files/11449689/Hardening.Kubeflow.Security.for.Enterprise.Environments.-.Julius.von.Kohout.DPDHL.Diana.Dimitrova.Atanasova.VMware.pdf)

File: security/README:.md
Comment (kimwnasptd) [inline security/README:.md:9]: Let's add links for 
1. The Calendar https://calendar.google.com/calendar/embed?src=kubeflow.org_7l5vnbn8suj2se10sen81d9428%40group.calendar.google.com
2. The drive with the recordings (link?)

**Review by kimwnasptd — COMMENTED** [review]
Did one more pass. 

Should we remove the `meetings` folder? We have a [gdoc](https://docs.google.com/document/d/1xGkg9GuO2OjvYhdONJFbSrpF66UKhtYonczttJoTv3s/edit#heading=h.fmin4de6g5bt) for the meeting notes so it would just add extra maintenance burden to copy the notes from the doc to GH every time. 

Instead I'd propose to update the `README.md` to point to all necessary information, like the gdoc, the calendar, drive etc

**Comment by akgraner** [comment]
Thanks everyone for all the feedback. Much appreciated. 
Here's what's been done. 

* Fixed all the empty folders
* Added the first iteration of the Security Team Policies and Procedures 
* Added the OWNERS file
* Deleted the Team Members folder and the Meetings folders
* Updated the meeting link 
* Updated the Notes link

@kimwnasptd and @james-jwu can you please review.  I think it should be good to go now.

File: security/policies-and-procedures.md
Comment (zijianjoy) [inline security/policies-and-procedures.md:13]: It is strongly recommended that we don't add a security tag to an open issue. 

You can find a security template put up by CNCF in https://github.com/cncf/tag-security/tree/main/project-resources.

Reference: https://github.com/cncf/tag-security/blob/main/project-resources/templates/SECURITY.md#reporting-a-vulnerability

Most CNCF projects right now use email to notify security issue. GitHub just released their private vuln reporting feature (https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability). Either email shared among the people who will be responsible for vuln response or this new GitHub reporting channel are fine.

Comment (akgraner) [reply security/policies-and-procedures.md:13]: @zijianjoy, I agree, but when I asked about getting an email created for the security team there was some push back. Since Google has to create the email address can you all create a security@kubeflow.org email for the team. The Working Groups asked that security issues related respective working groups be filed their, so (at the time) without being able to create an official email for the group and the resources we had available we came up with the above process. I'll take a look at the resources you have listed and rework the process and bring it up to the group on the 24th at our next meeting. 

In order to get this PR approved if I change this file to say we are currently working on the process and a link to our meeting notes would this work for sake of approving this?

**Comment by akgraner** [comment]
@james-jwu and @theadactyl  - in addition to the list from earlier this morning, I also removed the wording for the policy and procedures and will rework this based on @zijianjoy's suggestions above.  I removed @jbottum from owner's list.  Can this PR now be approved?

File: security/policies-and-procedures.md
Comment (james-jwu) [inline security/policies-and-procedures.md:9]: Typo: contributors

File: security/policies-and-procedures.md
Comment (zijianjoy) [reply security/policies-and-procedures.md]: We are in discussion about creating such email address. Make sense to track this effort in another PR. At the meantime, James Wu has LGTMed your PR here.

File: security/policies-and-procedures.md
Comment (akgraner) [reply security/policies-and-procedures.md:12]: @james-jwu  - Thank you! Fixed this typo.

File: security/policies-and-procedures.md
Comment (akgraner) [reply security/policies-and-procedures.md]: @james-jwu - Thank you! Fixed this typo.

**Comment by akgraner** [comment]
@james-jwu fixed the typos and the broken hyperlink which removed your lgtm. Sorry about that. Can you re-approve? Please and thank you.

**Pull request merged**
